src/Stats/sortFunction.jsx

```diff
@@ -0,0 +1,7 @@
+const sortDataByDate = (data) => {
+  const byDate = data.slice(0);
```

Comment:
I dont understand what is going on here, what is byDate? Why is it the first element in the array?